scripts/verify.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+#  Copyright 2018 The Kubernetes Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -e
+
+source ./common.sh
+
+header_text "go to the project root"
+cd ..
+
+header_text "install linter"
+fetch_go_linter
+
+header_text "running go mod tidy"
+go mod tidy
+
+header_text "running go vet"
+go vet ${MOD_OPT} ./...
+
+header_text "running go fmt"
+go fmt ${MOD_OPT} ./...
+
+header_text "running golangci-lint"
+golangci-lint run --disable-all \
+    --deadline 5m \
+    --enable=misspell \
+    --enable=structcheck \
+    --enable=deadcode \
+    --enable=errcheck \
+    --enable=varcheck \
+    --enable=unparam \
+    --enable=ineffassign \
+    --enable=nakedret \
+    --enable=interfacer \
+    --enable=misspell \
+    --enable=dupl \
+
+
+##todo(camilamacedo86): The following checks requires fixes in the code
+# --enable=golint
+# --enable=gocyclo
+# --enable=goimports
+# --enable=lll
+# --enable=goconst
+# --enable=gosec
+# --enable=maligned
+
```

Comment:
@Adirio, 

After merging this one, then we can do 1 PR to do the required fixes to enable each kind off test.